README.rst

```diff
@@ -44,6 +44,8 @@ Get to know
 
 - `Quick Start Guide </doc/Quick-Start-Guide.rst>`_
 
+- `Example Cluster Setup <https://github.com/Seagate/cortx-motr/discussions/285>`_
```

Comment:
I think the prefix part of the link can be omitted - `https://github.com/Seagate/cortx-motr`.

Reply:
Well, maybe not...

Reply:
It does not work. Reverted back.